Backend/src/controllers/auth.controller.ts

```diff
@@ -0,0 +1,102 @@
+import { Request, Response } from 'express';
```

Comment:
Add the missing license header referring JasmineGraph CPP files. Do this for all the files in this PR.

Reply:
sure

Reply:
Ok